doc_user/UserManual.adoc

```diff
@@ -167,7 +167,7 @@ NOTE: to apply the above *Exclusive Filters* one has to just type a __number__ a
 ** *Date*: “Date From” and “Date To” to select all visits occurring during the specified period
 ** *Age*: “Age From” and “Age To” to select patients by age
 ** *Type of patient*: Select visits by type of attendance: All / New / Re-Attendance
-** *User*: Select visits by doctor (USER column): only visits inserted by the logged user (me) or All
+** *User*: Select visits by a doctor (USER column): only visits inserted by the logged user (me) or All
```

Comment:
I'm not sure this is better, since the word "doctor" here is the search field...I might be wrong !